index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+    
+    <link rel="stylesheet" href="style.css">
+    <title>Capston</title>
+  </head>
+  <body>
+    <header>
+        <nav class="nav-bar">
+    
+            <i class="ham">
+                <i class="menuIcon fa fa-bars" ></i>
+                <i class="xIcon fa fa-close"></i>
+                </i>
+        
+                <ul class="menu">
+                  <li><a class="menuLink" href="index.html">Home</a></li>
+                  <li><a class="menuLink" href="#">Program</a></li>
+                  <li><a class="menuLink" href="about.html">About</a></li>
+                  <li><a class="menuLink" href="#">Join</a></li>
+                </ul>
+                
+                <ul class="topList">
+                    <li><i class="fa fa-facebook-f"></i></li>
+                    <li><i class="fa fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+
+        </nav>
+        <nav class="desktopNavFirst">
+            
+
+           <ul class="secondNavList">
+               <li><a href="./about.html">About</a></li>
+               <li>Program</li>
+               <li>Join</li>
+               <li>Sponsor</li>
+               <li>News</li>
+               <li><a href="./index.html">Summit</a></li>
+           </ul>
+          <a href="./index.html"> <img src="./images/images.png" alt="test"> </a>  
+        </nav>
```

Comment:
- [ ] You have a blank space at the top of the navbar, kindly remove it.
- [ ] Also, space out the logo and nav items, the logo should be to the extreme left.
- [ ] The last nav item should have a border around it, kindly add this.
- [ ] Also, use the correct background colour here, it shouldn't be black, it should be `#272A31`


_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124913352-df109000-dfa3-11eb-9456-88bd89532385.png) | ![image](https://user-images.githubusercontent.com/49621279/124913439-f8b1d780-dfa3-11eb-9e00-598dfaf2627d.png)